numpy/distutils/system_info.py

```diff
@@ -1911,31 +1911,36 @@ def calc_info(self):
             # Use the system BLAS from Accelerate or vecLib under OSX
             args = []
             link_args = []
-            if get_platform()[-4:] == 'i386' or 'intel' in get_platform() or \
+            arm = intel = 0
+            if 'arm64' in get_platform() or 'aarch64' in get_platform():
+                arm = 1
+            elif get_platform()[-4:] == 'i386' or 'intel' in get_platform() or \
                'x86_64' in get_platform() or \
                'i386' in platform.platform():
                 intel = 1
-            else:
-                intel = 0
             if (os.path.exists('/System/Library/Frameworks'
                               '/Accelerate.framework/') and
                     'accelerate' in libraries):
-                if intel:
+                if arm:
+                    pass
```

Comment:
Do we need an alternative here? We have Neon intrinsics, maybe they can be enabled?

Reply:
First I order the three architecture: the future ARM, the now X86, and the past PowerPC.

1. I've found no explicit options on clang to enable Neon intrinsics
```
xxx@MacBook-Pro numpy-1.16.6 % clang --help|grep neon       
xxx@MacBook-Pro numpy-1.16.6 % clang --help|grep simd
  -fopenmp-simd           Emit OpenMP code only for SIMD-based constructs.
xxx@MacBook-Pro numpy-1.16.6 % clang -v
Apple clang version 12.0.5 (clang-1205.0.22.9)
Target: arm64-apple-darwin20.5.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
```
maybe this apple clang default on Neon intrinsics, or maybe future clang will supply some explicit options?

gcc simd options is [here](https://gcc.gnu.org/onlinedocs/gcc-11.1.0/gcc/AArch64-Options.html#AArch64-Options)
`
‘simd’
Enable Advanced SIMD instructions. This also enables floating-point instructions. This is on by default for all possible values for options -march and -mcpu. 
`

2. should we ever support the PowerPC nowadays on macOS and the '-faltivec' option will cause warning on build:
`clang: error: the clang compiler does not support 'faltivec', please use -maltivec and include altivec.h explicitly`

[PowerPC cmdoption-clang-maltivec](https://clang.llvm.org/docs/ClangCommandLineReference.html#cmdoption-clang-maltivec)

replace '-faltivec' wiht '-maltivec':
[clang](https://gitlab.tu-berlin.de/daniel-schuermann/clang/commit/a776ed1122e48f9ade40c454678226303d8fab52)
[gcc](https://gcc.gnu.org/legacy-ml/gcc-patches/2004-05/msg01327.html)